<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <widget class="QLabel" name="secertIdLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>AccessKeyID:</string>
       </property>
      </widget>
      <widget class="QLabel" name="secertKeyLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>SecertAccessKey:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="secertIdLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="accountLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="secertKeyLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="nextButton_1">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>140</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="googleLoginPage">
      <widget class="QLineEdit" name="accountLineEdit_1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Private key:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="importButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>import</string>
       </property>
      </widget>
      <widget class="QLabel" name="jsonLabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nextButton_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>150</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="bucketPage">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create a default bucket:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bucketLineEdit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>54</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>110</y>
         <width>112</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Storage Class:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="storageClasscomboBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&amp;quot;standard&amp;quot; is for data frequently accessed,&amp;quot;standard-IA&amp;quot; is for data infrequently accessed with low cost and poor performance.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>STANDARD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STANDARD-IA</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>54</y>
         <width>112</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>BucketName:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="regionComboBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>82</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Asia</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>82</y>
         <width>112</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Region:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="preButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Pre</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextButton_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>170</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>51</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>secertIdLineEdit</tabstop>
  <tabstop>secertKeyLineEdit</tabstop>
  <tabstop>accountLineEdit</tabstop>
  <tabstop>nextButton_1</tabstop>
  <tabstop>accountLineEdit_1</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>nextButton_3</tabstop>
  <tabstop>bucketLineEdit</tabstop>
  <tabstop>regionComboBox</tabstop>
  <tabstop>storageClasscomboBox</tabstop>
  <tabstop>preButton</tabstop>
  <tabstop>nextButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
